homeassistant/components/deconz/deconz_event.py

```diff
@@ -11,19 +11,25 @@
 
 CONF_DECONZ_EVENT = "deconz_event"
 
+EVENT = "Event"
+
 
 async def async_setup_events(gateway) -> None:
     """Set up the deCONZ events."""
+    gateway.entities[EVENT] = set()
```

Comment:
The events are not entities.

Reply:
I realised they don't fit here when working on clearing up orphaned entities and devices

Reply:
We can probably get away with just renaming the attribute. Names are hard though.

Reply:
Everything regarding events are in gateway.events so I can probably just reference the information there

Reply:
Fixed in #40640